config.ini

```diff
@@ -57,13 +57,15 @@ include_css = static/d2l.css
 
 # The file used to post-process the generated tex file.
 post_latex = ./static/post_latex/main.py
+latex_url = https://d2l-webdata.s3.us-west-2.amazonaws.com/latex-styles/PT1.zip
 
 latex_logo = static/logo.png
 main_font = Source Serif Pro
 sans_font = Source Sans Pro
 mono_font = Inconsolata
 
 bibfile = d2l.bib
+style = default
```

Comment:
Can you add new entries `latex_url` and `style` to https://github.com/d2l-ai/d2l-book/blob/master/d2lbook/config_default.ini? Please set `style = default` in config_default.ini so we can remove this line.